api/auth.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const jwt = require("jsonwebtoken");
+
+module.exports = {
+  generateToken: function(user) {
```

Comment:
If you used your destructuring here w/ the param(I really picked it up from you), your payload could ultimately boil to  `    const payload = {
      id,
      username,
      department
    };` (You'd have to chance the `sub` key to `id` instead)